web/docs/command-line.md

```diff
@@ -37,9 +37,9 @@ tenzir-node [opts]
 tenzir-ctl [opts] cmd1 [opts1] cmd2 [opts2] ...
 ```
 
-Both long `--long=X` and short `-s X` exist. Boolean options do not require
-explicit specification of a value, and it suffices to write `--long` and `-s`
-to set an option to true.
+We have both long `--long=X` and short `-s X` options. Boolean options do not
+require explicit specification of a value, and it suffices to write `--long` and
+`-s` to set an option to true.
```

Comment:
You may want to mention that the `=` is mandatory for the long option and forbidden for short options, since this is an unusual quirk.